src/main/java/org/mule/extension/db/internal/source/RowListener.java

```diff
@@ -67,6 +68,7 @@
 @DisplayName("On Table Row")
 @Summary("Triggers a message per each row in a table")
 @Alias("listener")
+@Streaming
```

Comment:
In sources, you cannot make streamables repeatable because you have no access to the streaming helper, I dont think this is useful here.